go/porcelain/deploy.go

```diff
@@ -384,6 +394,57 @@ func (n *Netlify) WaitUntilDeployLive(ctx context.Context, d *models.Deploy) (*m
 	return n.waitForState(ctx, d, "ready")
 }
 
+func (n *Netlify) uploadFilesWithSkipSameFileOptimization(ctx context.Context, d *models.Deploy, files *deployFiles, observer DeployObserver, t uploadType, timeout time.Duration) error {
```

Comment:
What do you think of using `type` instead of `t` for the variable name?

Reply:
`type` is a reserved word in Go I believe. Unsure how well it would play with its compiler + I would think it could potentially be a bit confusing 🤔